pandas/core/resample.py

```diff
@@ -1404,15 +1404,18 @@ def __init__(
         self.fill_method = fill_method
         self.limit = limit
 
```

Comment:
can you edit the doc-string and add a versionupdate 1.2 tag (mention that end is added).

Reply:
sure